app/components/countdown_component.ts

```diff
@@ -0,0 +1,5 @@
+import { loadPolyfills } from '@18f/identity-polyfill';
+
+loadPolyfills(['custom-elements'])
+  .then(() => import('@18f/identity-countdown-element'))
```

Comment:
> Is there anything stopping us from doing this as a static import? Looking at the source, all it's doing is defining a subclass of `HTMLElement` which shouldn't need the `customElements` import/polyfill right?

I seem to recall that extending HTMLElement doesn't work in Internet Explorer without the polyfill. I will double-check, but: It's a good point to raise regardless, since there is a bit of a performance penalty from dynamically loading the element implementation which affects _all_ users. I'd rather try to limit any performance cost of the polyfill to the affected browsers.

While `loadPolyfills` is nice and clean and only loads the polyfills if and when they're needed, I wonder instead if we should just bundle every possible polyfill we need into a single pack, and load that pack in Internet Explorer before any other script. It's a bit wasteful since not every polyfill is needed on every page, but I feel more comfortable with the graceful degradation affecting only Internet Explorer, vs. this overhead impacting every user.

Reply:
Yeah unsure if we want to mix in that polyfill refactor in this PR but if IE is literally the only browser we need our polyfills for, that definitely seems like a decent tradeoff

Reply:
For follow-up: #6060